spring-web/src/main/java/org/springframework/http/HttpHeaders.java

```diff
@@ -16,32 +16,20 @@
 
 package org.springframework.http;
 
+import org.springframework.util.Assert;
```

Comment:
You can refer  [Spring-Framework-Code-Style](https://github.com/spring-projects/spring-framework/wiki/Spring-Framework-Code-Style#import-statements) for the ordering of import statements.